trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilitySnapshot.test.js

```diff
@@ -86,8 +86,6 @@ describe('mapFacility', () => {
 
     const expectedFacilityValueExportCurrency = `${mockCurrency.id} ${formattedFacilityValue}`;
 
-    const facilityStage = mapBssEwcsFacilityStage(mockFacilityStage, mockFacilityTfm.facilityStage);
```

Comment:
How come this has been removed?

Reply:
Removed following a discussion with Tony on a separate thread about the unit tests for this file. But as a consequence of that discussion, I've realised that this is not needed at all (either in the test or in the original `mapFacilitySnapshot` file): 

- In `mapFacilitySnapshot` we clone the facility snapshot and add additional properties to the cloned object. This `clonedFacility` is then passed into several functions where the fields on the cloned facility are used and accessed. However `facilityStage` is not used / accessed in any of these functions, it's only used as the value for the new `facilityStage` field on the `mappedFacilitySnapshot`. So I've refactored here to calculate the `facilityStage` just when it's needed on the mappedSnapshot & not add it as a temporary property to the cloned snapshot. 

Not sure if that's a clear explanation or not, happy to hop on a call to show you!